packages/components/popover/src/popover.ts

```diff
@@ -57,6 +57,9 @@ export const popoverProps = buildProps({
     type: Boolean,
     default: true,
   },
+  'onUpdate:visible': {
+    type: Function as PropType<(visible: boolean) => void>,
+  },
```

Comment:
<img width="892" alt="image" src="https://user-images.githubusercontent.com/39730999/178522726-15660561-8fc3-41fe-b0c7-1c958d5787cb.png">
It seems to be repeated here.

Reply:
Here I read what is written in the `tooltip`, and I will test whether the deletion will affect it.

Reply:
Confirmation is required, otherwise it cannot be judged whether visible is a bidirectional attribute.

Reply:
```ts
import { useAttrs } from 'vue'

const attrs = useAttrs()
```

Maybe you can use `attrs.onUpdate:visible` to determine whether it is passed in

Reply:
let me try.

Reply:
You need to remove `onUpdate:visible` in `props` first

Reply:
It is operated on the premise of removal.

Reply:
![image](https://user-images.githubusercontent.com/23251408/178539525-26023dab-7cea-441b-a067-211225fbcd43.png)

Reply:
![image](https://user-images.githubusercontent.com/23251408/178544636-80eac7eb-2410-483d-aea7-4f08449e291d.png)